plugins/services/src/js/containers/service-connection/ServiceConnectionEndpointList.tsx

```diff
@@ -43,26 +43,14 @@ export default class ServiceConnectionEndpointList extends React.Component<{
     return getDisplayValue(protocol);
   }
 
-  getHostPortValue(portDefinition) {
-    if (!portDefinition.hostPort) {
-      return <Trans render="span">Auto Assigned</Trans>;
-    }
-
-    let hostPortValue = portDefinition.hostPort;
-
-    if (!hostPortValue) {
-      hostPortValue = portDefinition.port;
-    }
-
-    if (hostPortValue) {
-      return (
-        <EndpointClipboardTrigger
-          command={getDisplayValue(hostPortValue.toString())}
-        />
-      );
-    }
-
-    return getDisplayValue(hostPortValue);
+  getHostPortValue({ hostPort }) {
+    return hostPort ? (
+      <EndpointClipboardTrigger
+        command={getDisplayValue(hostPort.toString())}
+      />
+    ) : (
+      <Trans render="span">Auto Assigned</Trans>
+    );
```

Comment:
@janisz changed this here as well.

Reply:
?

Reply:
[`1f10fdc` (#4945)](https://github.com/dcos/dcos-ui/pull/4945/commits/1f10fdcaaf6cd6e9a3f0995fadbfe90615d6a277)

Reply:
But there is a fallback to `port` I think it will not be clear

Reply:
```
  getHostPortValue({ hostPort, port }) {
    return !hostPort ? (
      <Trans render="span">Auto Assigned</Trans>
    ) : (
      <EndpointClipboardTrigger command={getDisplayValue((hostPort || port).toString())}/>
    );
  }
```